coderd/workspacebuilds.go

```diff
@@ -581,10 +581,12 @@ func (api *API) notifyWorkspaceUpdated(
 // @Produce json
 // @Tags Builds
 // @Param workspacebuild path string true "Workspace build ID"
+// @Param expect_status query string false "Expected status of the job" Enums(running, pending)
```

Comment:
It might pay to have a description field (idk if our API generation has something like that, you might need to check other routes) that explains in more detail what this actually does:

> If `expect_status` is supplied, the request will be rejected with 499 Precondition Failed if the job doesn't match the state when performing the cancellation.